### PR TITLE
Update cfgov-sheer-templates dependency

### PR DESCRIPTION
**Pull request by Scotchester**
Had to update the shrinkwrap to get it to install the latest cfgov-sheer-templates (2.1.5). This will ensure that the header and footer styling is correct post-CF4 upgrade.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,16 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
+- Update cfgov-sheer-templates dependency to 2.1.5 to support cfgov-refresh's
+  update to Capital Framework 4.
+
+## [2.0.2] - 2017-06-19
+### Changed
 - Modify `@grid_wrapper-width` to match Capital Framework
+- Update the About Us section to the updated text and call to action
+
+### Fixed
+- jQuery global reference
```
Comment (anselmbradford) [inline CHANGELOG.md]: What was done here?

**Comment by anselmbradford** [comment]
Looks good, just be sure the Changelog changes are how you want them.

File: CHANGELOG.md (hunk 1 above)
Comment (Scotchester) [inline CHANGELOG.md]: This includes changes from previous PRs that weren't added to the changelog. All I've done here is what is now in the Unreleased section.

**Comment by Scotchester** [comment]
To clarify further: After merging this, I'll manually push an update to package.json and CHANGELOG.md to make this the v2.0.3 release.

**Comment by sebworks** [comment]
@Scotchester, what version of NPM are you using?

**Comment by Scotchester** [comment]
@sebworks 3.10.10

**Comment by sebworks** [comment]
ok, just wondering why the shrinkwrap file got so small. It must be the new format.

**Comment by Scotchester** [comment]
Hmm... that is kind of odd. It looks like maybe subdependencies are not being added? Should I have run something other than just `npm shrinkwrap` to create the file?

**Comment by sebworks** [comment]
@Scotchester, what version of node are you running?

**Comment by Scotchester** [comment]
@sebworks 6.11.0

**Comment by sebworks** [comment]
ooooh ok, it's only shrinkwrapping the dependencies, which is correct. The OAH package.json file is a little different than cfgov-refresh.

In cfgov-refresh we list all our build dependencies as `dependencies` and not `devDependencies`. So `npm shrinkwrap --dev` should work.  

I'm interested to see if this will fail on Jenkins as it currently exists.

**Comment by Scotchester** [comment]
Shouldn't it still include subdepdendencies of those regular dependencies, though?

**Comment by sebworks** [comment]
looks good, 👍

**Comment by Scotchester** [comment]
> Shouldn't it still include subdepdendencies of those regular dependencies, though?

Responding to my own query here, for the record: It actually _was_ including subdependencies, there were just very very few of them for the project's prod dependencies.

**Pull request merged**